### PR TITLE
Add instructions for different Solr modules.

### PR DESCRIPTION
**Pull request by MacWeber**

### CODE REVIEW

File: src/frameworks/drupal7/solr/apachesolr-module.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Using Solr with the module Apache Solr Search on Drupal 7.x
+
+This page is about configuring Solr with the module [Apache Solr Search](https://www.drupal.org/project/apachesolr). If your project uses [Search API](https://www.drupal.org/project/search_api) then you should follow the instructions [here](search-api-module.md).
```
Comment (Crell) [inline src/frameworks/drupal7/solr/apachesolr-module.md]: Let's avoid "here" links.  Instead use "see the [Search API]() documentation.

File: src/frameworks/drupal7/solr/search-api-module.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
-# Using Solr with Drupal 7.x
+# Using Solr with the module Search API on Drupal 7.x
 
-## Requirements
+This page is about configuring Solr with the module [Search API](https://www.drupal.org/project/search_api). If your project uses [Apache Solr Search](https://www.drupal.org/project/apachesolr) then you should follow the instructions [here](apachesolr-module.md).
```
Comment (Crell) [inline src/frameworks/drupal7/solr/search-api-module.md]: As above, let's reword this to avoid "here links".

File: src/frameworks/drupal7/solr/solr.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Using Solr with Drupal 7.x
```
Comment (Crell) [inline src/frameworks/drupal7/solr/solr.md]: This file doesn't really add much.  Let's just drop it and make the other two files top-level in this section.  Fewer pages for users to click through.

File: src/frameworks/drupal7/solr/apachesolr-module.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+```php
+if (isset($_ENV['PLATFORM_RELATIONSHIPS'])) {
+  $relationships = json_decode(base64_decode($_ENV['PLATFORM_RELATIONSHIPS']), TRUE);
+
+  if (!empty($relationships['solr'])) {
+    // Override search API server settings fetched from default configuration.
+    foreach ($relationships['solr'] as $endpoint) {
+      // Replace the machine name of the server here.
+      $environment_machine_name = 'MACHINE_NAME_OF_SOLR_SERVER';
```
Comment (Crell) [inline src/frameworks/drupal7/solr/apachesolr-module.md]: Inline comments FTW.

**Comment by Crell** [comment]
@MacWeber Do you want to continue this or should I take it over?  I'm good either way.

**Comment by MacWeber** [comment]
Sorry @Crell, I got busy with other stuff. I will take it again on Wednesday.

**Comment by MacWeber** [comment]
@Crell I had pinged you on Slack, but I think you might have missed the message. All suggested changes are now applied.

**Comment by Crell** [comment]
Spiffy! Thanks, @MacWeber!
